bluepyopt/ephys/efeatures.py

```diff
@@ -426,6 +426,12 @@ def calculate_feature(
         from .extra_features_utils import calculate_features
 
         """Calculate feature value"""
+        if peak_times is None:
+            if return_waveforms:
+                return None, None
+            else:
+                return None
+            
```

Comment:
ci says there are blank spaces here. could you remove them?

Reply:
Done. Thanks.